inc/3rd-party/plugins/sumome.php

```diff
@@ -0,0 +1,47 @@
+<?php 
+/**
+ * Compatibility with SumoMe
```

Comment:
We should add more details about the reason for this compatibility here, in case we forget why we did this 😄

Reply:
All done :+1: